index.js

```diff
@@ -1,9 +1,28 @@
-var _ = require('lodash');
+var index = {}
 
-var data = require('./codes-postaux.json');
+// Index by postalCode value
+require('./communes2016.json').forEach(function (commune) {
+  commune.codesPostaux.forEach(function (codePostal) {
+    if (!(codePostal in index)) {
+      index[codePostal] = [];
+    }
+    index[codePostal].push({
+      nomCommune: commune.nom,
+      codeInsee: commune.code,
+      population: commune.population || 0
+    });
+  });
+});
 
-var index = _.groupBy(data, 'codePostal');
+// Sort by population
+for (var codePostal in index) {
+  index[codePostal].sort(function (a, b) {
+    if (a.population <= b.population) return 1;
+    if (a.population >= b.population) return -1;
+    return 0;
```

Comment:
IMO that sorting should be done either lazily, or as preprocessing before publishing the package. I find it stupid that we add a large computation for all clients on each first `require`.